Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    ln -snf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && echo $CONTAINER_TIMEZONE > /etc/timezone
+
+RUN apt-get install -y curl git openjdk-8-jdk python3 python3-pip unzip wget && \
+    ln -s /usr/bin/python3 /usr/bin/python
+
+RUN wget https://downloads.openmicroscopy.org/bio-formats/6.6.1/artifacts/bftools.zip && \
+    unzip bftools.zip && \
+    cp bftools/* /usr/local/bin/
+
+RUN pip install vitessce==0.1.0a11
```

Comment:
Do you think it's a good idea to have `scimap` as well?

Reply:
I had wondered if you want to bundle them together. It makes sense since they are both tools in this repo, but vitessce is also pretty stand-alone from scimap. Whatever you prefer is fine with me.

Reply:
>vitessce is also pretty stand-alone from scimap.

Sounds reasonable. In that case, we will need another docker image for `scimap`.

Reply:
Agree, I can build one next week.